lib/global-styles.php

```diff
@@ -171,22 +171,32 @@ function gutenberg_experimental_global_styles_get_theme() {
  * @return string CSS rule.
  */
 function gutenberg_experimental_global_styles_resolver( $global_styles ) {
-	$css_rule = '';
-
-	$token    = '--';
-	$prefix   = '--wp' . $token;
-	$css_vars = gutenberg_experimental_global_styles_get_css_vars( $global_styles, $prefix, $token );
-	if ( empty( $css_vars ) ) {
-		return $css_rule;
-	}
+	$css_rules = '';
+
+	// This is only for demo purposes, it should be pluggable
+	// so new selectors can be register
+	// (3rd party blocks, custom selectors, etc).
+	$selectors = array(
+		'core'                      => ':root',
+		'core/paragraph'            => '.wp-site-blocks p',
```

Comment:
Should we try to move this to a more stable version e.g: make this selectors part of block.json so which block can configure its selector?